sqlalchemy_ibmi/base.py

```diff
@@ -784,7 +788,14 @@ def create_connect_args(self, url):
             raise ValueError("Option entered not valid for "
                              "IBM i Access ODBC Driver")
 
-        name_setting = '1' if opts.get('use_system_naming', 'True') == 'True' else '0'
+        try:
+            name_setting = '0' if not util.strtobool(
+                opts['use_system_naming']) else '1'
```

Comment:
```suggestion
            name_setting = str(util.strtobool(opts['use_system_naming']))
```

Confusingly, strtobool returns 1/0, not True/False, so we can convert the value directly to a string using the str function